portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,259 @@
+                    <button type="button" class="btn btn-warning" onclick="addQuote()"><i class="bi bi-chat-square-quote text-light"></i></button>
+                    <p class="pt-2" id="quote"></p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Contact me section -->
+    <section class="p-5 text-center" id="contactMe">
+        <div class="container">
+            <h1 class="text-center pb-3">Contact Me :D</h1>
+            <p class="lead">Here's some sites where you can follow or contact me. <br> Please I'm desperate.</p>
+            <div class="row text-center g-4">
+                <!-- GitHub card -->
+                <div class="col-md d-flex align-items-stretch">
+                    <div class="card bg-light text-dark">
+                        <div class="card-body text-center">
+                            <div class="h1 mb-3">
+                                <i class="bi bi-linkedin"></i>
+                            </div>
+                            <h3 class="card-title">
+                                Linkedin
+                            </h3>
+                            <p class="card-text">
+                                I just recently made a Linkedin account, so its pretty empty right now. Still I would love to make connections.
+                            </p>
+                            <button href="https://www.linkedin.com/in/mateo-herrera-lavalle-6059a9239" role="button" class="btn btn-outline-primary">Go Here</button>
+                        </div>
+                    </div>
+                </div>
+                <!-- Linkedin card -->
+                <div class="col-md d-flex align-items-stretch">
+                    <div class="card bg-light text-dark">
+                        <div class="card-body text-center">
+                            <div class="h1 mb-3">
+                                <i class="bi bi-github"></i>
+                            </div>
+                            <h3 class="card-title">
+                                GitHub
+                            </h3>
+                            <p class="card-text">
+                                Here is my GitHub account. Theres repo's for schoolwork and personal projects.
+                            </p>
+                            <button href="https://github.com/MateoHerrera0" role="button" class="btn btn-outline-primary">Go There</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- <p>Click <a href="/hello">here</a> to see content created by server code.</p> -->
+    
+    <script src="script.js"></script>
+    <!-- Bootsrap javascript CDN -->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
```

Comment:
Should this be at the top?

Reply:
I put it at the bottom so all elements were loaded before I tried calling them in functions, I thought you were supposed to do it this way. Should I put it at the top?

Reply:
I'm no Web Dev expert so not sure what the best pattern is but I've usually seen these up top unless the script file needs to wait for the elements to load.